src/cbc_sdk/endpoint_standard/base.py

```diff
@@ -616,3 +697,297 @@ def _search(self, start=0, rows=0):
 
             log.debug("current: {}, total_results: {}".format(current, self._total_results))
 
+
+class EnrichedEventFacetQuery(Query, AsyncQueryMixin):
```

Comment:
This class could also use methods for adding criteria and exclusions. Platform Queries have methods for these that could be copied into this Query Class https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/ec2bb7d16d61755c71e30653c3a813ccf0f76460/src/cbc_sdk/enterprise_edr/base.py#L390

Reply:
I noticed those are implemented in your FacetQuery base class, I guess inheriting from it once it gets merged would by itself solve that problem?